go.mod

```diff
@@ -5,6 +5,7 @@ go 1.16
 replace (
 	github.com/go-logr/logr => github.com/go-logr/logr v0.1.0
 	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.3.1 // indirect
+	github.com/k14s/kbld => github.com/anujc25/carvel-kbld v0.31.0-update-vendir
```

Comment:
Using `v0.31.0` with `vendir` version updated to `v0.23.0`. It is not possible to build the CLI without this change because another carvel dependency requires `vendir` version to be `v0.23.0`.

Also created PR here: https://github.com/vmware-tanzu/carvel-kbld/pull/188

Reply:
Updated the `go.mod` reference as the PR got merged.